tsc/meetings/2023-07-25.md

```diff
@@ -0,0 +1,78 @@
+Minutes from OpenVDB TSC meeting, July 25th, 2023
```

Comment:
I've decided to omit the meeting count from now on because it's becoming impossible to follow exactly

Reply:
If we keep the shared doc of who last wrote the meetings up to date, we also can track this number.